monero-outreach-docs/translations/es/storyboard/refuge_es.md

```diff
@@ -0,0 +1,94 @@
+ 
+Cabe anotar que en ciertos estados de los EEUU existe el llamado ["Equitable Sharing" o intercambio equitativo](https://www.justice.gov/criminal-mlars/equitable-sharing-program), un programa el cual procede a distribuir entre el estado y las autoridades federales los activos liquidados a través de la incautación y el decomiso. En el 2010, más de [$500 millones de USD fueron compartidos entre oficiales y agencias](https://www.wsj.com/articles/SB10001424053111903596904576514583716653122) a través del programa de intercambio equitativo, con de más de $5 billones de USD distribuidos desde la inauguración del programa en 1984; un "[slush fund](https://www.merriam-webster.com/dictionary/slush%20fund)" (fondo o dádiva para sobornar al "investido", político). Si estás creyendo que estos fondos son ejercidos solamente para los "avances" de las actividades políticas, puedes estar muy equivocado. Los funcionarios y oficiales encargados de las políticas públicas han utilizado estos fondos de "intercambio equitativo" para [subsidiar comidas, fiestas costosas y gastos personales](https://harvardlawreview.org/2018/06/how-crime-pays-the-unconstitutionality-of-modern-civil-asset-forfeiture-as-a-tool-of-criminal-law-enforcement/). 
+ 
+Para la persona promedio sometidas a los bloques legales de los derechos civiles, el recurso de apelación es difícil de conseguir. Y, el incentivo para excederse listo para aquellos dentro de esos ámbitos legales. 
+
+### **Tres clases de decomiso civil en los EEUU**
+
+**Caso 1:** En el 2013 un [hombre en el estado de Mississippi fue detenido por un policía local](https://www.vice.com/en_ca/article/9bnezd/bad-cop-blotter-asset-forfeiture-the-cash-cow-of-the-drug-war) para una revisión de tráfico rutinaria, que se convirtió en todo menos en una revisión de rutina. Después de realizar una inspección al vehículo, el oficial encontró un compartimiento que contenía $360.000 USD. Ya sabrán que sucedió después. El dinero fue confiscado sin haber encontrado evidencias de haber crimen. 
+
+**Caso 2:** En el mismo año (2013), Alda Gentile y su hijo fueron detenidos por exceso de velocidad (su hijo estaba conduciendo). [La policía de Georgia reviso el vehículo en búsqueda de drogas](https://www.marketplace.org/2016/04/06/world/controversial-civil-forfeiture-program-back-action) y finalmente encuentran e incautan $11,530 USD de los cuales Gentile les explico que era para comprar una casa en la Florida. A Gentile no la acusaron y le devolvieron el dinero una semana después. 

```

Comment:
In my understanding, the sentence here refers to:

+ _**Gentile nunca fue acusada de algún crímen**_, y le devolvieron el dinero una semana después.

In allusion to her not ever being charged of a crime before (was never charged with a crime). The translation is not wrong at all. I understand in the translation that her wasn't charged with a crime in that exact moment, not before. What do you think? Either way this is not wrong, it's just a comment.

Reply:
Don't worry, is always good to build the translation between several contributors, in this case us. I like when you give suggestions, this helps build a better language between the workgroup and find better definitions in future translations. Thank you as always >:).
When I we don't agree on something you know I always call it out :see_no_evil:.

Reply:
Done >:).